.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         run: yarn prettier-check
 
       - name: Build TerriaJS tests
-        run: yarn gulp lint release --continue
+        run: yarn gulp lint build --continue
```

Comment:
Is this change intentional, or was it to speed up the CI loop during development of the PR?

Reply:
Yeah the new version of React doesn't allow running tests on a production build